docs/user/azure/customization.md

```diff
@@ -0,0 +1,41 @@
+# Azure Platform Customization
+
+The following options are available when using Azure:
+
+- `machines.platform.azure.type` - the VM instance type
+- `platform.azure.region` - the Azure region (location) that the installer will use when creating resource group and resources
+
+## Examples
+
+An example `install-config.yaml` is shown below. This configuration has been modified to show the customization that is possible via the install config.
+
+```yaml
+apiVersion: v1
+baseDomain: example.com
+controlPlane:
+  name: master
+  platform:
+    aws:
```

Comment:
probably `aws` -> `azure` (and below) ;)

Reply:
https://github.com/openshift/installer/compare/fc6532c8637a2c37f4c0a028da1a5a430ac19302...3bb0cfefaf513837f94ff94c1bc59e047421323b